src/test/java/de/fearnixx/jeak/test/plugin/ControllerTestPlugin.java

```diff
@@ -15,34 +16,22 @@
 @JeakBotPlugin(id = "controllertestplugin")
 public class ControllerTestPlugin extends AbstractTestPlugin {
 
-    @Inject
-    @Config
-    public IConfig configRef;
-
-    @Inject
-    public IDataCache dataCache;
-
     @Inject
     IRestControllerService restControllerService;
 
 
     public ControllerTestPlugin() {
         super();
-        addTest("test_injected_configRef");
-        addTest("test_injected_dataCache");
     }
 
     @Listener
     public void onInitialize(IBotStateEvent.IInitializeEvent event) {
-        if (configRef != null) {
-            success("test_injected_configRef");
-        }
-
-        if (dataCache != null) {
-            success("test_injected_dataCache");
-        }
         restControllerService.registerController(TestController.class, new TestController());
```

Comment:
Feel free to add a "register_successful" test here. If adding tests will not work anytime for some reason, we will notice it here.